### ISSUE TITLE
NTR: AICA-riboside high in CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8324]
New term request AICA-riboside high in CSF (elthat_130330192416): 5-amino-4-imidazolecarboxyamide; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000238
- label: Elevated CSF 5-amino-4-imidazolecarboxyamide concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: The concentration of 5-amino-4-imidazolecarboxyamide in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8324
- synonyms: Elevated CSF AICA-riboside concentration(NONE)
